### DISCUSSION TITLE
How to use?

### DISCUSSION THREAD

**Discussion by dirgalaks** [thatskymod/Sky-CotL-Scripts#49] [General]
how to use and download the cheat apps?

**Comment by thatskymod** [comment]
@dirgalaks, the repository offers a variety of scripts and mods tailored to different audiences, ranging from beginner to advanced. Each "app" or "cheat" serves a distinct purpose and has its own setup procedure.

Would you like guidance on a specific app, or are you seeking general information?

**Reply by dirgalaks** [reply]
i need a cheat for candle run, and maybe you can guide me how to use it

**Reply by thatskymod** [reply]
Understood. Here's a high-level overview matching your request:

### - Option #1: Virtual Space + GameGuardian + Lua script(s).
I will skip this for now, as it involves some in-depth setup and technical knowledge. **GameGuardian** is a very versatile and powerful game hacking and memory editing engine, but it's probably not the most ideal if you're just starting out or simply wish something that works "out of the box".

### - Option #2: Canvas
**Canvas** is a cheating app specifically for Sky and is designed to be very user-friendly. It only takes a sole ~20 MB of your phone's storage and can modify different aspects of the game (including automatic candle running as you mentioned). There's an always up-to-date tutorial on how to install and configure Canvas in the [Getting Started](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/README.md#getting-started) section of the app.

### Special Note:
Canvas works **only for** ![Android](https://img.shields.io/badge/Android-3DDC84?style=flat&logo=android&logoColor=white), and there's nothing we can change about it. If you're on ![iOS](https://img.shields.io/badge/iOS-000000?style=flat&logo=ios&logoColor=white), you might consider other options.

Apart from the recommended AutoWax file in the README, feel free to navigate to the [Libs](https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/Canvas/Libs) folder and try out some of the other mods Canvas has to offer! You can use the Download button in Github's dropdown menu to save the file(s) locally.

That's it for now. Reach out if you have further questions or inquiries!
